.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: '1.13'
+      - name: Set up go env
+        run: |
+          echo "::set-env name=GOPATH::$(go env GOPATH)"
+          echo "::add-path::$(go env GOPATH)/bin"
+        shell: bash
+      - name: Install jq
+        run: |
+          mkdir -p deps/bin
```

Comment:
I believe jq can be installed via apt-get in Ubuntu

Reply:
We had chosen to hard code our version in our existing CI, I wasn't sure whether or not that was important, so carried over the practice. Happy to explore aptitude installation if we think that's a preference.